src/object.d

```diff
@@ -38,6 +38,67 @@ alias dstring = immutable(dchar)[];
 
 version (D_ObjectiveC) public import core.attribute : selector;
 
+// Size used to store the TypeInfo at the end of an allocation for structs that
+// have a destructor
+private size_t structTypeInfoSize(T)() pure nothrow @nogc
+{
+    // can't use because it's in rt.lifetime
```

Comment:
What about this? Any solutions?

Reply:
The simplest would be to make `callStructDtorsDuringGC` `package` and import `rt.lifetime`.

Reply:
Can't import `rt.lifetime`. You can't import anything from `rt/*` since it's not in `mak/COPY`'s `$(IMPDIR)`.

Maybe I'm missing something, as the GC is able [to import](https://github.com/dlang/druntime/blob/master/src/gc/impl/conservative/gc.d#L41) from `rt/util/`

Reply:
Then you can do a hack, declare an `extern` variable and use `pragma(mangle)` to get the same mangling as for `callStructDtorsDuringGC` declared in `rt.lifetime`.

Reply:
We can do even better. Here's an example from the existing code:

```d
import core.internal.traits : externDFunc;
alias rt_tlsgc_init = externDFunc!("rt.tlsgc.init", void* function() nothrow @nogc);
```

Reply:
Does that work with non-function types?

Reply:
You could add an `extern (C)` function in lifetime.d to return `callStructDtorsDuringGC` and then just declare that prototype in object.d.  You can already find precedent for that in object.d (e.g. https://github.com/dlang/druntime/blob/4ad638f61a9b4a98d8ed6eb9f9429c0ef6afc8e3/src/object.d#L3909-L3910)